src/main/java/taxi/controller/car/SearchDriverByCarController.java

```diff
@@ -0,0 +1,28 @@
+package taxi.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.DriverService;
+
+public class SearchDriverByCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long carId = Long.parseLong(req.getParameter("car_id"));
+        List<Driver> drivers = driverService.getAll();
```

Comment:
let's remove this controller at all
reason - because you are fetching all drivers from db - this will be bad for performance
you could add this functionality if you performed correct SQL query - but then we need to make changes on dao and service levels as well